core/scripts/chaincli/cliex/flags.go

```diff
@@ -0,0 +1,16 @@
+package cliex
+
+import (
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// BindStringFlag binds the given string flag.
+func BindStringFlag(cmd *cobra.Command, key, flag, short, usage string) {
```

Comment:
Where is this used? Or is it implicit? Can't find any calls to it in this PR.

Reply:
Yup, forgot to remove. Fixed.